mkdocs.yml

```
@@ -185,9 +185,11 @@ nav:
         - "Diagram mode: Application Swimlane": admin/features/process-design/swimlane-for-applications/swimlane-for-applications.md
         - "Method: Global responsible role for sub processes": admin/features/process-design/global-responsible-role/global-responsible-role.md
       - Repository design:
+        - "Method: Additional relationships for documents": admin/features/repository-design/additional-relations-documents/additional-relations-documents.md
         - "Method: System ArchiMate Converting": admin/features/repository-design/archimate-conversions/archimate-conversions.md
         - "Method: Classification of information for Documents": admin/features/repository-design/classification-of-info/classification-of-information.md
-        - "Method: Maintain Measures status": admin/features/repository-design/measure-status/measure-status.md
+        - "Method: 'Documents to be considered' for organizational units": admin/features/repository-design/documents-org-units/documents-org-units.md
+          "Method: Maintain Measures status": admin/features/repository-design/measure-status/measure-status.md
```

```suggestion
          - "Method: Maintain Measures status": admin/features/repository-design/measure-status/measure-status.md
```

Otherwise it will break the build process since the line is not recognized as a list entry.